frontend/src/App.js

```diff
@@ -7,9 +7,12 @@ import SearchInputs from "./pages/SearchInputs";
 import Properties from "./pages/Properties";
 import Login from "./pages/Login";
 import PropertyDetails from "./pages/PropertyDetails";
-
+import propertyResponse from "./properties.json";
 
 export default function App() {
+
+  const properties = propertyResponse;
+
   return (
       <ThemeProvider theme={stylesheet}>
```

Comment:
very generic name, might be worth considering finding a name that expresses what styles get affected by this theme